postgres-appliance/Dockerfile

```diff
@@ -53,30 +53,28 @@ WORKDIR $PGHOME
 RUN pip install patroni==$PATRONIVERSION
 
 # WAL-e requests a relatively new version of requests, install other modules via pip as well
-RUN pip install requests wal-e --upgrade
+RUN pip install requests wal-e --upgrade pystache
 
```

Comment:
Should we move `--upgrade` before the first package name?  It would be easier to understand, IMO.